Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -51,25 +54,61 @@ class NetKDRequestDevice : public EmulationDevice
     return std::nullopt;
   }
 
+  enum class CurrentFunction : u32
+  {
+    None,
+    Account,
+    Check,
+    Receive,
+    _,
+    Send,
+    Save,
+    Download,
+  };
```

Comment:
You can do `None = 0, Account = 1` etc here to avoid the undefined value in the middle here.